mysqld_exporter.go

```diff
@@ -178,6 +178,17 @@ func main() {
 		}
 	}
 	http.HandleFunc(*metricPath, prometheus.InstrumentHandlerFunc("metrics", newHandler(enabledScrapers)))
+	// add an endpoint for mysql health state
+	http.HandleFunc("/-/healthy", func(w http.ResponseWriter, r *http.Request) {
+
+		if collector.GetmysqlHealth() {
+			w.Write([]byte("ok: mysqldup"))
```

Comment:
To match the metric we provide[0], maybe call it `mysql_up`

[0]: https://github.com/prometheus/mysqld_exporter/blob/master/collector/exporter.go#L100-L104